ark/utils/segmentation_utils.py

```diff
@@ -127,3 +129,55 @@ def concatenate_csv(base_dir, csv_files, column_name="point", column_values=None
             combined_data = pd.concat((combined_data, temp_data), axis=0, ignore_index=True)
 
     combined_data.to_csv(os.path.join(base_dir, "combined_data.csv"), index=False)
+
+
+def visualize_watershed_transform(segmentation_labels_xr, channel_data_xr,
+                                  output_dir,
+                                  overlay_channels, fovs=None,
+                                  save_tifs='overlays'):
+    """Runs the watershed transform over a set of probability masks output by deepcell network
+    Saves xarray to output directory
+    Args:
+        segmentation_labels_xr (xarray.DataArray): xarray containing segmentation labels
+        channel_data_xr (xarray.DataArray): xarray containing TIFs
+        output_dir (str): path to directory where the output will be saved
+        overlay_channels (tuple): channels to overlay segmentation output over
+        fovs (list): field of view
+        save_tifs (str): flag to control what level of output to save. Must be one of:
+        all - saves all tifs
+        overlays - saves color overlays and segmentation masks
```

Comment:
@alex-l-kong is this formatted correctly for rtd?

Reply:
No, it's not. I looked at the most recent ReadTheDocs for this branch and the docstring for visualize_watershed_transform is messed up. @kaiwenliu @wang0324 please look at the `mode` arg description in `create_paired_xarray_fovs` of `test_utils.py` to see how to properly format a list in a Google docstring.

Looks like the lack of a newline between the description and the Args section is the main reason why the docstring is not formatted correctly. Please fix that too. Even if ReadTheDocs builds correctly, it's still important to see what it builds because we're still working on adding more docstring checks to docs/conf.py.